src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -1,4 +1,4 @@
-# --------------------------------------------------------------------------------------------
+﻿# --------------------------------------------------------------------------------------------
```

Comment:
I guess @viananth is editing this file on Windows, therefore a BOM is added. BOM is not recommended according to Unicode standard.

> 2.6 Encoding Schemes
>
> ... Use of a BOM is neither required nor recommended for UTF-8, but may be encountered in contexts where UTF-8 data is converted from other encoding forms that use a BOM or where the BOM is used as a UTF-8 signature. See the “Byte Order Mark” subsection in Section 16.8, Specials, for more information.

You don't have to mend this file this time. It's overall harmless, and we can take it out with a script. But please avoid this in the future. Here's an instruction about avoiding adding BOM to the file in VS: https://vlasovstudio.com/fix-file-encoding/.